.github/workflows/lynx.yml

```diff
@@ -0,0 +1,58 @@
+name: '🔎 Lynx Testnet Example'
+
+on:
+  schedule:
+    - cron: "0/20 * * * *"  # Every 20 minutes
+  pull_request:
+    branches:
+      - main
+      - development
+      - epic-*
+  workflow_dispatch:
+
+env:
+  DEMO_L1_PROVIDER_URI: ${{ vars.DEMO_L1_PROVIDER_URI }}
+  DEMO_L2_PROVIDER_URI: ${{ vars.DEMO_L2_PROVIDER_URI }}
+
+jobs:
+  python-tests:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.10" ]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install latest Rust stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+
+      - id: setup_python
+        name: Set up Python ${{ matrix.python-version }} Environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - id: python_cache
+        name: Retrieve Cached Python Dependencies
+        uses: actions/cache@v3
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ runner.os }}-pip-${{ steps.setup_python.outputs.python-version }}-${{ hashFiles('dev-requirements.txt', 'requirements.txt') }}
+
+      - name: Install Dependencies
+        if: steps.python_cache.outputs.cache-hit != 'true'
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[dev]
```

Comment:
@cygnusv, FYI this step will be modified - see #3272 .

Reply:
ACK. Let's wait for 3272 then, because I have a question there. Whatever we agree, I'll make the proper change here.

Reply:
Example of a failure that #3272 should resolve is the current failure from this PR python-tests run. https://github.com/nucypher/nucypher/actions/runs/6466815165/job/17555592260?pr=3274.

Without #3272 , you would need to delete the cache for PR 3274 from https://github.com/nucypher/nucypher/actions/caches and rerun the python-tests job.